src/style.css

```
@@ -1 +1,82 @@
-/* Styles go here */
+body {
+  margin: 0;
+  background: #e5e5e5;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+.logo {
+  display: inline-flex;
+}
+
+.logo__link {
+  display: inline-flex;
+  padding: 10px 0;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  display: inline-flex;
+  font-family: Roboto, sans-serif;
+  text-transform: uppercase;
+  font-size: 12px;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+
+  justify-content: center;
+  text-align: center;
+  align-items: center;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  position: relative;
+  text-decoration: none;
+  color: #000;
+  padding: 23px 0;
+
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__link:active::after {
```

```suggestion
.is-active::after {
```
You can use is-active class instead